pkgs/development/python-modules/craft-archives/default.nix

```
@@ -43,6 +44,7 @@ buildPythonPackage rec {
     lazr-restfulclient
     overrides
     pydantic
+    debian
```

```suggestion
    python-debian
```
Please rename this package to what it's called on PyPI.

---

Just so I understand, you mean add another commit that actually renames the package in nixpkgs? Or do something differently here?

Looks like the pname is already set to `python-debian`, it's just in a directory named `debian`, not `python-debian`.

---

> Just so I understand, you mean add another commit that actually renames the package in nixpkgs?

Exactly.



> Looks like the pname is already set to `python-debian`, it's just in a directory named `debian`, not `python-debian`.

More importantly, the attribute in `python-packages.nix` needs to be renamed.